<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="row0col0">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="row0col1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="row0col2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="row1col0_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>190</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="row1col0">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="row1col1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="row2col2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>350</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="row2col1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="row2col0">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1053</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
